src/components/ui/iconbutton/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { FaAngleRight } from "react-icons/fa";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine